code-for-canada/project-thundercat
# [New eMIB Entry Process] Combination of new functionalities from my last PRs

# Description

This PR introduced the following new functionalities combined  together:
- only one one page is displayed when you click on '_Star the eMIB Sample Test_' instead of having 5 pages of instruction.
- after clicking on '_Enter eMIB_' and then confirm in the popup box, you'll get to the instruction part of the test.
- the '**Background**' and '**Inbox**' tabs will be disabled.
- there will be only one button at bottom right in the test footer to start the test.
- once ready to start the test, after clicking on '_Start test_' and confirm in the popup box, the disabled tabs will become enabled.
- at the same time, the test footer will show the same two buttons we had before (Quit test and Submit test buttons).

## Type of change

- New feature (non-breaking change which adds functionality)

## Screenshot

**New eMIB Entry Process**
---
**Starting the _eMIB Sample Test_:**

![image](https://user-images.githubusercontent.com/23021242/56908070-9a0f7b80-6a73-11e9-8470-7ea55d15f524.png)

**Enter eMIB:**

![image](https://user-images.githubusercontent.com/23021242/56908151-c4613900-6a73-11e9-818e-ed84080df889.png)

**Enter eMIB confirmation:**

![image](https://user-images.githubusercontent.com/23021242/56908201-dc38bd00-6a73-11e9-9b49-17561773309d.png)

**Show the instruction tab first + _Background_ and _Inbox_ tabs are disabled + Only _Start test_ button in the test footer:**

![image](https://user-images.githubusercontent.com/23021242/56908259-f377aa80-6a73-11e9-9b99-9b22fe46e41a.png)

**Start test confirmation**

![image](https://user-images.githubusercontent.com/23021242/56908339-1e61fe80-6a74-11e9-9b0e-36b1d6dddf99.png)

**Disabled tabs become enabled + _Quit test_ and _Submit test_ are now displayed in test footer**

![image](https://user-images.githubusercontent.com/23021242/56908496-6c770200-6a74-11e9-8366-b2ceccb9441c.png)

# Testing

Manual steps to reproduce this functionality:

1.  Start the eMIB Sample Test.
2.  Follow the new entry process and make sure it looks as expected.
3.  Make also sure that it doesn't break anything.

# Checklist

Applicable for all code changes.
